build/Dockerfile

```diff
@@ -1,3 +1,22 @@
+# Copyright (c) 2019 InfraCloud Technologies
```

Comment:
Can you please update the year 2019-2020 in source files as well ?

Reply:
@sanketsudake year in the source files license headers should point to the year it was added. e.g https://github.com/infracloudio/botkube/pull/266/files#diff-c30adc0ccd728f66fd5cc8b794cab804R1. That's we have observed in other OSS projects as well
cc: @bhavin192